module4/assignment4.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Assignment #4</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<link rel="stylesheet" href="_css/assignment4theme.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<!-- Ted Pysh
+	index.html
+	info2439
+	Thoendel
+	9/8/19 
+	-->
+
+
+</head>
+<body>
+<div data-role="page">
+	<div data-role="header">
+	<h1>Assignment #4 </h1> <p><a href="index.html">Back to main page</a></p>
+	</div> <!--Closing header div -->
+	<div role="main" class="ui-content">
+		<form action="assignment4.php" method="post">
```

Comment:
This form action indicates that you have your own php on the web server ready to process submissions...the assignment had you pointing to a pre-written php script. This is why your form doesn't write to the db.